kb/disorders/CHIME_syndrome.yaml

```diff
@@ -0,0 +1,183 @@
+name: CHIME_syndrome
+description: A rare autosomal recessive neuroectodermal disorder characterized by colobomas, congenital heart defects, ichthyosiform dermatosis, intellectual disability, and ear anomalies caused by mutations in the PIGL gene affecting glycosylphosphatidylinositol (GPI) anchor biosynthesis.
+category: Genetic
+parents:
+- Neuroectodermal Disorder
+synonyms:
+- Zunich neuroectodermal syndrome
+- Coloboma-heart-ichthyosis-mental retardation-ear syndrome
+prevalence:
+- population: Global
+  percentage: 0.000001
+pathophysiology:
+- name: PIGL Gene Mutations
+  description: Mutations in PIGL gene disrupt glycosylphosphatidylinositol (GPI) anchor biosynthesis. PIGL encodes a de-N-acetylase required for GPI anchor formation, essential for anchoring various cell surface proteins including receptors, enzymes, and adhesion molecules.
+  biological_processes:
+  - preferred_term: glycosylphosphatidylinositol biosynthetic process
+    term:
+      id: GO:0006506
+      label: GPI anchor biosynthetic process
+  evidence:
+  - reference: PMID:22444671
+    supports: SUPPORT
+    snippet: "CHIME syndrome is characterized by colobomas, heart defects, ichthyosiform dermatosis, mental retardation (intellectual disability), and ear anomalies, including conductive hearing loss. Whole-exome sequencing on five previously reported cases identified PIGL, the de-N-acetylase required for glycosylphosphatidylinositol (GPI) anchor formation, as a strong candidate."
+    explanation: Establishes PIGL mutations as the cause of CHIME syndrome and describes the gene's role in GPI anchor biosynthesis.
+- name: GPI Anchor Deficiency
+  description: Reduced levels of GPI-anchored proteins on cell surface due to impaired biosynthesis, affecting normal cellular functions and signaling pathways. This leads to the multisystem phenotype characteristic of CHIME syndrome.
+  evidence:
+  - reference: PMID:22444671
+    supports: SUPPORT
+    snippet: "Furthermore, cell lines derived from these cases had significantly reduced levels of the two GPI anchor markers, CD59 and a GPI-binding toxin, aerolysin (FLAER), confirming the pathogenicity of the mutations."
+    explanation: Demonstrates functional consequences of PIGL mutations showing reduced GPI anchor markers in patient cells.
+phenotypes:
+- category: Ophthalmic
```

Comment:
agent: the HPO term is more specific; it may still be appropriate but this should be reflected in the description